app/Http/Controllers/UserController.php

```diff
@@ -12,10 +14,27 @@ class UserController extends Controller
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Request $request)
     {
         //
-    }
+        $users = [];
+        try {
+            $users = User::all();
+            $users = json_decode(json_encode($users),true);
+        } catch (Exception  $e) {
+            // log error
+        }
+
+        $perPage = 10;
```

Comment:
Why are you not using Laravel's built in paginator?
`App\User::paginate(10)`
Learn more here https://laravel.com/docs/5.4/pagination#paginating-eloquent-results

Reply:
> Why are you not using Laravel's built in paginator?
> `App\User::paginate(10)`
> Learn more here https://laravel.com/docs/7.x/pagination#paginating-eloquent-results

hmm this is the same laravel paginator specified in the documentation, only implemented a different way because of the try-catch block https://laravel.com/docs/7.x/pagination#displaying-pagination-results